packages/Statie/src/FileSystem/FileFinder.php

```diff
@@ -35,7 +35,8 @@ public function findLatteLayoutsAndSnippets(string $directory): array
     public function findInDirectoryForGenerator(string $directoryPath): array
     {
         $directoryInfo = new NativeSplFileInfo($directoryPath);
-        $pathPattern = '#' . preg_quote($directoryInfo->getFilename() . DIRECTORY_SEPARATOR, '#') . '#';
+        $path = $this->normalizePath($directoryInfo->getFilename() . DIRECTORY_SEPARATOR);
+        $pathPattern = '#' . preg_quote($path, '#') . '#';
```

Comment:
Btw, is this `preg_quote` still needed, when `normalizePath` handles those slashes?

Reply:
Ask yourself if it were any different if it was "do I really need to escape this variable in the template if it comes from the server and it's just a path?". 

I think you could break it using a dot or square brackets in the path (not sure if that's valid though).

Reply:
I see, it makes sense